Diseño template Students

MaksiConGit opened the pull request:
Diseño de template funcional. Falta show.

MaksiConGit committed:
feat<back-front>: se añaden bocetos de diseño de asistencia y materias

MaksiConGit committed:
feat<back-front>: se añaden bocetos de diseño de asistencia y materias

MaksiConGit committed:
feat<back-front>: se añaden bocetos de diseño de asistencia y materias

MaksiConGit committed:
feat<back-front>: se añaden bocetos de diseño de asistencia y materias

MaksiConGit committed:
feat<back-front>: se añaden bocetos de diseño de asistencia y materias

MaksiConGit committed:
feat<back-front>: se añaden bocetos de diseño de asistencia y materias

MaksiConGit committed:
feat<back-front>: se añaden bocetos de diseño de asistencia y materias

MaksiConGit committed:
feat<back-front>: se añaden bocetos de diseño de asistencia y materias

MaksiConGit committed:
feat<back-front>: se añaden bocetos de diseño de asistencia y materias

MaksiConGit committed:
feat<back-front>: se añaden bocetos de diseño de asistencia y materias

Marzopax committed:
feat<front>: horarios + tablas

tablas editables

Marzopax committed:
feat<front>: horarios + tablas

tablas editables

Marzopax committed:
feat<front>: horarios + tablas

tablas editables

Marzopax committed:
feat<front>: horarios + tablas

tablas editables

Marzopax committed:
feat<front>: horarios + tablas

tablas editables

Marzopax committed:
feat<front>: horarios + tablas

tablas editables

Marzopax committed:
feat<front>: horarios + tablas

tablas editables

Marzopax committed:
feat<front>: horarios + tablas

tablas editables

Marzopax committed:
feat<front>: horarios + tablas

tablas editables

Marzopax committed:
feat<front>: horarios + tablas

tablas editables

EstefaniSaiquita committed:
feat<front>: se crea componentes, views de profesores y alumnos

EstefaniSaiquita committed:
feat<front>: se crea componentes, views de profesores y alumnos

EstefaniSaiquita committed:
feat<front>: se crea componentes, views de profesores y alumnos

EstefaniSaiquita committed:
feat<front>: se crea componentes, views de profesores y alumnos

EstefaniSaiquita committed:
feat<front>: se crea componentes, views de profesores y alumnos

EstefaniSaiquita committed:
feat<front>: se crea componentes, views de profesores y alumnos

EstefaniSaiquita committed:
feat<front>: se crea componentes, views de profesores y alumnos

EstefaniSaiquita committed:
feat<front>: se crea componentes, views de profesores y alumnos

EstefaniSaiquita committed:
feat<front>: se crea componentes, views de profesores y alumnos

Marzopax committed:
feat<front>: arreglo de template

el index esta, la view intermedia que tiene el editar y eliminar no y el editar esta a la mitad

Marzopax committed:
feat<front>: arreglo de template

el index esta, la view intermedia que tiene el editar y eliminar no y el editar esta a la mitad

EstefaniSaiquita committed:
<front> view de material hecho

EstefaniSaiquita committed:
<front> view de material hecho

EstefaniSaiquita committed:
<front> view de material hecho

EstefaniSaiquita committed:
<front> view de material hecho

EstefaniSaiquita committed:
<front> view de material hecho

EstefaniSaiquita committed:
<front> view de material hecho

EstefaniSaiquita committed:
<front> view de material hecho

Joako343 committed:
fix<front> : Create, Edit y Show

Se puede crear, editar y mostrar alumno/s

Joako343 committed:
fix<front> : Create, Edit y Show

Se puede crear, editar y mostrar alumno/s

Joako343 committed:
fix<front>: Eliminacion de code

Elimine codigo comentado

Joako343 committed:
fix<front>: Eliminacion de code

Elimine codigo comentado

MaksiConGit committed:
Merge branch '154-diseño-template-students' of https://github.com/MaksiConGit/proyecto-final-tercero into 154-diseño-template-students

MaksiConGit committed:
feat<back-front>: index, edit, create funcional con diseño

MaksiConGit committed:
feat<back-front>: index, edit, create funcional con diseño

MaksiConGit committed:
feat<back-front>: index, edit, create funcional con diseño

MaksiConGit committed:
feat<back-front>: index, edit, create funcional con diseño

MaksiConGit committed:
feat<back-front>: index, edit, create funcional con diseño

MaksiConGit committed:
feat<back-front>: index, edit, create funcional con diseño

Joako343 committed:
fix<front>: Componente creado

se creo el componente para el show

Joako343 committed:
fix<front>: Ya no uso el componente xd

En el propio show ya se muestra todo, falta arreglar el boton de volver y datos del backend

MaksiConGit committed:
Merge branch '154-diseño-template-students' of https://github.com/MaksiConGit/proyecto-final-tercero into 154-diseño-template-students

LucasDLabo committed:
Implementado Livewire, Roles y Permisos, Login

 - Se instaló Livewire.
 -  Se instaló Spatie Roles y Permisos.
 - Se crearon migraciones polimórficas.
 - Se implemento el login dentro del template.
 - Se crearon usuarios con datos reales para directivo, profesor y estudiante.

LucasDLabo committed:
feat: Botón de login/register y Perfil

 - Se implementó un condicional para logearse/registrarse o mostrar el perfil si ya se esta logeado.

LucasDLabo committed:
fix: user_id de Student Seeder

LucasDLabo reviewed (changes requested):
**Felicitaciones por el trabajo**
Posibles cambios a realizar (Algunas son ideas por lo que pueden no hacerse o ser de baja prioridad):
### Student Index
- [ ] Si es una lista general, mostrar todas los alumnos paginados y permitir filtrar por carrera y curso.
- [x] Idea: Si la card expresa "Usuario: Sin asignar" , mostrar un botón para cada uno que lleve a la asignación.
- [x] Idea: Para mejorar la experiencia de usuario, agregar un efecto de hover al pasar por encima del user del Estudiante para saber que es clickeable.
- [x] Idea: Para mejorar la experiencia de usuario, agregar un efecto de hover al pasar por encima de un Estudiante.
- [ ] No esta listado en la navbar de la izquierda la sección de "Estudiantes".
- [ ] Idea: Implementar en la lista de Estudiantes, opciones para editar y eliminar sin tener que acceder a cada estudiante con su respectivo mensaje de confirmación para evitar misslicks.
- [ ] Idea: El botón de crear Estudiantes puede pasar un poco desapercibido.
### Student Create/Edit
- [x] El create/edit debe permitir elegir Institución, Carrera, Curso para crear/editar el estudiante y su relación correspondiente con course_students.
- [x] En el create/edit falta poder seleccionar País y Provincia.
- [ ] El botón de create/edit dice 'enviar' en vez de crear/editar.
- [ ] Idea: Al momento de crear un Estudiante, redirigir al show del mismo.
- [ ] Los iconos de los inputs debe estar acorde a lo que pide el input.
- [x] El usuario debera crearse automaticamente.
- [x] El Student Edit tiene como título "Crear Profesor".
### Student Show
- [ ] Falta implementar el show de Estudiante, que debe permitir ver todos sus cursos, sus notas y su asistencia.
- [ ]  El botón de eliminar no tiene ninguna confirmación lo que puede llevar a missclicks.
- [x] El titulo dice "Profesor:"
- [x] La ciudad siempre aparece como "No asignada" a pesar de si tener una.
- [ ] Idea: Implementar un botón para volver hacia atrás.
### General
- [ ]  El breadcrum deberá funcionar correctamente en todas las páginas.
- [ ] El titulo de la pestaña debe estar acorde a la vista.

MaksiConGit committed:
Merge branch '148-diseo-template-user' of https://github.com/MaksiConGit/proyecto-final-tercero into 148-diseo-template-user

MaksiConGit committed:
feat<back-front>: edit, create, index funcional con diseño

LucasDLabo committed:
feat: Create Student

 - Se implemento que el usuario de student se cree automáticamente.
 - Se implemento podes elegir y asignarle muchos cursos a un estudiante.
 - Ahora el create de student permite elegir País, Provincia y Ciudad

LucasDLabo committed:
feat: Create Student

 - Se implemento que el usuario de student se cree automáticamente.
 - Se implemento podes elegir y asignarle muchos cursos a un estudiante.
 - Ahora el create de student permite elegir País, Provincia y Ciudad

LucasDLabo committed:
Merge branch '154-diseño-template-students' of https://github.com/MaksiConGit/proyecto-final-tercero into 154-diseño-template-students

LucasDLabo committed:
feat: Edit Student con multiple asignación de cursos

 - Se implemento poder editar el alumno a la vez que editar sus cursos asignados.
 - Se creo la migración de "logindata"

LucasDLabo committed:
feat: Edit Student con multiple asignación de cursos

 - Se implemento poder editar el alumno a la vez que editar sus cursos asignados.
 - Se creo la migración de "logindata"

LucasDLabo committed:
feat: Paginación de Students index

 - Se paginó de a 5 students el index

LucasDLabo committed:
feat: Paginación de Students index

 - Se paginó de a 5 students el index

LucasDLabo committed:
feat: Student Show cursos y exámenes

 - Se implemento poder ver los cursos que tiene asociado un estudiante junto con sus exámenes.

LucasDLabo committed:
feat: Student Show cursos y exámenes

 - Se implemento poder ver los cursos que tiene asociado un estudiante junto con sus exámenes.

LucasDLabo committed:
feat: Filtro de student por institution, career y course

 - Se implementó un filtro a tiempo real para listar alumnos entre institución, carreras y cursos

LucasDLabo committed:
feat: Filtro de student por institution, career y course

 - Se implementó un filtro a tiempo real para listar alumnos entre institución, carreras y cursos

LucasDLabo committed:
feat: Student Index Search

 - Se implementó una barra buscadora para encontrar los alumnos.

LucasDLabo committed:
feat: Student Index Search

 - Se implementó una barra buscadora para encontrar los alumnos.

LucasDLabo committed:
fix: Bug exámenes de cursos no asignados

 - Se corrigió un bug que mostraba examenes de cursos que no correspondian al student.

LucasDLabo committed:
fix: Bug exámenes de cursos no asignados

 - Se corrigió un bug que mostraba examenes de cursos que no correspondian al student.

MaksiConGit committed:
feat<back-front>: index y show de students con diseño y datos agregados

MaksiConGit committed:
fix<back>: error con los roles en create

MaksiConGit committed:
Merge branch '154-diseño-template-students' of https://github.com/MaksiConGit/proyecto-final-tercero into 154-diseño-template-students

MaksiConGit committed:
fix: breadcrumb y título del archivo

MaksiConGit committed:
feat<back>: aside funcional con diseño

LucasDLabo reviewed (dismissed):
confio

MaksiConGit committed:
fix: se arregla componente aside

MaksiConGit committed:
feat<front>: se añade aside al resto de students
